maven/bnd-maven-plugin/src/site/site.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since we don't build or publish a maven site, I don't think this should be part of the source.